nixos/tests/firefox.nix

```diff
@@ -1,5 +1,14 @@
-import ./make-test-python.nix ({ pkgs, firefoxPackage, ... }: {
-  name = "firefox";
+import ./make-test-python.nix ({ pkgs, firefoxPackage, ... }:
+let firefoxPackage' = firefoxPackage.override (args: {
+      extraPrefsFiles = (args.extraPrefsFiles or []) ++ [
```

Comment:
Ouch, does this rebuild firefox? 😅

Reply:
Only the wrapper, not the software itself. So this will finish pretty much instantly.